code/modules/mob/living/carbon/human/life.dm

```diff
@@ -181,8 +181,9 @@ var/global/list/organ_damage_overlays = list(
 	pulse = handle_pulse()
 	for(var/obj/item/weapon/grab/G in src)
 		G.process()
-	/*if(mind && mind.vampire)
-		handle_vampire()*/
+	var/datum/role/vampire/V = isvampire(src)
```

Comment:
NO.

Roles have a process that's called every game tick. Use that. We're making role datums to avoid this.

Reply:
That's a much better idea